src/stories/components/ChartPopover/ChartPopover.story.tsx

```diff
@@ -121,13 +121,7 @@ const MinWidth = bindWithProps(ChartPopoverSvgStory);
 MinWidth.args = { children: dialogContent, width: 'auto', minWidth: 250 };
 
 const OnOpenChange = bindWithProps(ChartPopoverSvgStory);
-OnOpenChange.args = {
-	children: dialogContent,
-	width: 'auto',
-	onOpenChange: (isOpen) => {
-		console.log('isOpen', isOpen);
-	},
-};
+OnOpenChange.args = { children: dialogContent, width: 'auto' };
```

Comment:
Don't want a `console.log`. I fixed the actions addon so it will display the onOpenChange stuff for us!